src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -0,0 +1,49 @@
+package nextstep.subway.ui;
+
+import nextstep.subway.application.LineService;
+import nextstep.subway.dto.LineCreateRequest;
+import nextstep.subway.dto.LineResponse;
+import nextstep.subway.dto.LineUpdateRequest;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createStation(@RequestBody LineCreateRequest lineRequest) {
+        LineResponse line = lineService.saveLine(lineRequest);
+        return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
+    }
+
+    @GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        return ResponseEntity.ok().body(lineService.findAllLines());
+    }
+
+    @GetMapping(value = "/lines/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> findLine(@PathVariable Long id) {
+        return ResponseEntity.ok().body(lineService.findLine(id));
+    }
+
+    @PutMapping(value = "/lines/{id}")
+    public ResponseEntity updateStation(@PathVariable Long id, @RequestBody LineUpdateRequest lineUpdateRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Void> updateStation(@PathVariable Long id, @RequestBody LineUpdateRequest lineUpdateRequest) {
```
제네릭 타입은 형을 명시하는 것이 좋습니다!

Reply:
여지껏 이렇게 살아왔습니다.. 죄송합니다. 감사합니다...!